index.html

```diff
@@ -421,7 +422,7 @@ <h4 data-i18n="index.page.requirements.updates.header">Frequent Updates</h4>
         unsupported.os = "Debian 7";
         unsupported.i18nKey = "debian-7";
         unsupported.url = "https://github.com/adobe/brackets/issues/4816";
-        unsupported.default = "Unfortunately, we currently don't support Debian 7. <a href='" + unsupported.url + "'>Take a look at the GitHub issue</a> for further information.";
+        unsupported.defaultText = "Unfortunately, we currently don't support Debian 7. <a href='" + unsupported.url + "'>Take a look at the GitHub issue</a> for further information.";
```

Comment:
@Mark-Simulacrum I see your logic, but as a native English speaker and one who strives to use proper English, especially in website design, "don't currently" just does not work. The original "We currently [do not] support Debian 7" was better, in both understanding and grammar (though in personal web design, I eliminate contractions completely).